benchmark/benchmark17_test.go

```diff
@@ -0,0 +1,57 @@
+// +build go1.7
+
+package benchmark
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/benchmark/stats"
+)
+
+func BenchClient(b *testing.B, maxConcurrentCalls, reqSize, respSize int, runMode, enableTrace string) {
+	if strings.EqualFold(enableTrace, "Tracing") {
+		grpc.EnableTracing = true
+	} else {
+		grpc.EnableTracing = false
+	}
+	if strings.EqualFold(runMode, "Unary") {
+		runUnary(b, maxConcurrentCalls, reqSize, respSize)
+	} else {
+		runStream(b, maxConcurrentCalls, reqSize, respSize)
+	}
+}
+
+func BenchmarkClient(b *testing.B) {
+	modeTrace := []struct {
+		runMode     string
+		enableTrace string
+	}{
+		{"Unary", "Tracing"},
+		{"Unary", "noTrace"},
+		{"Stream", "Tracing"},
+		{"Stream", "noTrace"},
+	}
+	maxConcurrentCalls := []int{1, 8, 64, 512}
+	reqSize := []int{1, 1024}
```

Comment:
Nit: reqSizeBytes and respSizeBytes

It took digging through like 5 calls to figure out whether this was bytes or KB.  Since it's bytes, 1MB (1024*1024) is going to be more interesting than 1KB.

FWIW, this is also just a field in a proto message, so "0" actually means there is still some data being transmitted.